### DISCUSSION TITLE
Agenda for the Flatcar Developer Sync on Wednesday, 23rd of October 2024, 2:30pm UTC

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1549] [Flatcar Developer Sync]
In our monthly Flatcar developer syncs maintainers review development / roadmap progress, discuss special projects like OS composability, secure / trusted boot, vendor support, and CNCF incubation status. We provide an overview of ongoing work, give an outlook of future work, and plan the next release.

Join if you're curious about current or future Flatcar development and/or to raise visibility of a feature, bug, or PR.

- Jitsi Link: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  https://www.youtube.com/live/R3Sgi49R4oI
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Jitsi call. Please introduce yourself and share what brings you here today.

### Docker 26 is now in Beta (4054.1.0), please test!
<details><summary>A number of legacy features have been removed by upstream.</summary>

  [Removed features](https://github.com/docker/cli/blob/master/docs/deprecated.md):
  - [Devicemapper storage driver](https://github.com/docker/cli/blob/master/docs/deprecated.md#device-mapper-storage-driver)
  - [Pushing and pulling with image manifest v2 schema 1](https://github.com/docker/cli/blob/master/docs/deprecated.md#pushing-and-pulling-with-image-manifest-v2-schema-1)
  - [OOM-score adjust for the daemon](https://github.com/docker/cli/blob/master/docs/deprecated.md#oom-score-adjust-for-the-daemon)
  - [logentries logging driver](https://github.com/docker/cli/blob/master/docs/deprecated.md#logentries-logging-driver)
  - [IsAutomated field, and "is-automated" filter on docker search](https://github.com/docker/cli/blob/master/docs/deprecated.md#isautomated-field-and-is-automated-filter-on-docker-search)
  - [Container short ID in network Aliases field](https://github.com/docker/cli/blob/master/docs/deprecated.md#container-short-id-in-network-aliases-field)
  - [legacy API versions](https://github.com/docker/cli/blob/master/docs/deprecated.md#deprecate-legacy-api-versions)
  - [Container and ContainerConfig fields in Image inspect](https://github.com/docker/cli/blob/master/docs/deprecated.md#container-and-containerconfig-fields-in-image-inspect)

</details>

## Monthly Contributions and Advisory status Review
1. Discuss the latest [Monthly Contributions Report](https://github.com/flatcar/Flatcar/issues?q=is%3Aissue+is%3Aopen+Monthly+contributions+report+label%3Akind%2Fmetric)
   * First-Time Contributors in last month:
     * @fhemberger - improvements to sysext build documentation, k3s and rke2 Kubernetes version bumps ([changes](https://github.com/flatcar/sysext-bakery/commits?author=fhemberger&since=2024-09-22&until=2024-10-21))
2. Review of last month's advisory stats in the latest [Contributions Report](https://github.com/flatcar/Flatcar/issues?q=is%3Aissue+is%3Aopen+Monthly+contributions+report+label%3Akind%2Fmetric)
   a. Advisory status and planning: https://github.com/orgs/flatcar/projects/7/views/17

## Ongoing and Planned Tasks Review

1. Roadmap review: https://github.com/orgs/flatcar/projects/7/views/9 (@t-lo, @miao0miao )
2. Projects status review:
   a. Composable OS: sysext and ClusterAPI (@tormath1) - https://github.com/orgs/flatcar/projects/7/views/14
   b. SecureBoot / Trusted Boot (@sayanchowdhury ) - https://github.com/orgs/flatcar/projects/7/views/20
   c. CNCF Incubation process (@miao0miao) - https://github.com/orgs/flatcar/projects/7/views/18
   d. New vendors / platforms developments (@tormath1) - https://github.com/flatcar/Flatcar/issues?q=is%3Aopen+is%3Aissue+label%3Aplatform%2Fnew-platform
4. Maintainer team tactical / implementation (aka "line work") status review (@sayanchowdhury): https://github.com/orgs/flatcar/projects/7/views/1
5. Open Mic: Work to assign / User issues and PRs to discuss?

### Release Planning
- Ongoing and upcoming releases tracker: (https://github.com/orgs/flatcar/projects/7/views/8) (@sayanchowdhury)
- Talk about the upcoming release and the status of the packages
- Status of the ongoing release
- Preparation of the next release 

### Q&A
- Questions from community participants, answered by the Flatcar maintainers 

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar developer sync: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-developer-sync
- [x] Close this discussion as "Resolved".

**Comment by tormath1** [comment]
I won't join this sync. Here's my update:
* On CAPI / sysext side:
  * resumed the work on CAPZ + Flatcar
  * new Flatcar support incoming for Cluster API Proxmox provider (https://github.com/ionos-cloud/cluster-api-provider-proxmox/issues/302)
* On the vendors side:
  * got a request to support Ionos Cloud with a few contributions already proposed (https://github.com/flatcar/Flatcar/issues/1570)
